proposals/lip-0049.md

```diff
@@ -368,13 +404,17 @@ The role of the channel terminated message is to inform chains that their chain
 
 This message is empty. The `params` schema for channel terminated is the empty schema.
 
-##### Creating Channel Terminated Messages
+##### Creation
 
 A channel terminated message is created by the Interoperability module when [terminating a chain account][lip-0045#terminateChain], for example when encountering a malicious cross-chain update transaction or malicious cross-chain message.
```

Comment:
> malicious cross-chain update transaction or malicious cross-chain message.

It is not well-defined what malicious should mean and who is actually malicious.

Reply:
edited